tech/languages/csharp/dotnet-installation.md

```diff
@@ -4,18 +4,29 @@ subsection: csharp
 order: 3
 ---
 
-# .NET SDK installation
+# .NET Core installation
```

Comment:
I think the page's title should be consistent with what's in "title" in jekyll header (please change one of them).